tests/blobcache.bats

```diff
@@ -55,6 +55,37 @@ function _check_matches() {
         expect_output --from="$unmatched" "$5"  "$6 should not match"
 }
 
+# Integration test for https://github.com/containers/image/pull/1645
+@test "blobcache: blobs must be reused when pushing across registry" {
+	start_registry
+	run_buildah login --tls-verify=false --authfile ${TEST_SCRATCH_DIR}/test.auth --username testuser --password testpassword localhost:${REGISTRY_PORT}
+	run_buildah build $WITH_POLICY_JSON -t exp -f $BUDFILES/simple-multi-step/Containerfile
+	run_buildah push $WITH_POLICY_JSON --format docker exp dir:${TEST_SCRATCH_DIR}/exp
```

Comment:
(Is the `--format docker` doing something nonobvious here? It seems to me that it should neither help nor hurt, so I’m wondering whether I am missing something that makes it necessary.)

Reply:
This is not needed I can remove this.